### ISSUE TITLE
BB and simulation explorers are not displayed

### ISSUE THREAD

**Issue by EnricaMez** [Open-Systems-Pharmacology/MoBi#322]
Hi all,
I'm now having problems with displaying BB and simulation explorers, as well as history manager in MoBi 7.2.2. "Show or Hide" option from the Views ribbon is not working.

I tried also re-installing OSP suite 7.2.2 and working with both portable versions 7.2.2 and 7.3. But this didn't solve the issue.

**Comment by Yuri05** [comment]
Please try the following (in case of MoBi 7.3 **NON-portable** version):
* close MoBi
* remove the file "C:\Users\ **_USERID_**\AppData\Roaming\Open Systems Pharmacology\MoBi\7.3\Settings.xml" (replace **_USERID_** with your windows user name)
* start MoBi again

Did it help?

**Comment by StephanSchaller** [comment]
That's a glitch in the user settings that does rarely happen.

It can be solved by deleting the AppData in the User folder.

If I remember correctly all will be reset if you delete the UserSettings.xml (Be aware that all other USer Settings in the SOftware will also be reset to default).

UserSettings.xml is located at

C:\Users\ <your Name> \AppData\Roaming\Open Systems Pharmacology\ <PK-Sim or MoBi> \ <7.2 or 7.3>

**Comment by StephanSchaller** [comment]
;-)

**Comment by PavelBal** [comment]
@Yuri05 @msevestre We urgently need to create a FAQ section and post the solution to this problem here!

**Comment by msevestre** [comment]
Yeah. Also add a button to clear the settings like we do in PKSim. This should really take 1 hour to implement
